### ISSUE TITLE
Identifier 'blazorGoogleMapsDateFormat' has already been declared.

### ISSUE THREAD

**Issue by Curlack** [rungwiroon/BlazorGoogleMaps#223]
I'm getting the following error when running the `ClientSideDemo` project:
![namespace_bug](https://user-images.githubusercontent.com/298525/204575561-8da91059-742c-47d4-803d-aac754eeed0d.png)

I think we should move all of the javascript into its own namespace e.g. `window.blazorGoogleMaps` as to not pollute the global namespace or have "bleeding" into the javascript of other packages.

**Comment by valentasm1** [comment]
Does latest 2.4.2 nuget solve issue?
https://www.nuget.org/packages/BlazorGoogleMaps/2.4.2

**Comment by Curlack** [comment]
No, it didn't. I was using the latest code when I encountered these issues.

**Comment by valentasm1** [comment]
Please make PR then.

**Comment by Curlack** [comment]
Ok now I'm leaving my comfort zone. I'll make the PR, just need to "quickly" learn it :)
Also not sure when to use PR or what I've done now.
Just to confirm I understand what I've done now: Did I make a fork and requested a merge into the master branch?

**Comment by Curlack** [comment]
I think I got it. These tools doing things for us in the background...lol. I did it previously, just didn't know it. PR on the way. Thanks.

**Comment by Curlack** [comment]
I did it! Victory! :D

**Comment by Curlack** [comment]
The changes to `objectManager.js` looks vicious, in short, I only move the entire script into `blazorGoogleMaps` object that is returned by `IIFE` (immediately invoked function expression). I did this so we can encapsulate private members. Then I moved the `directionService` handler (previously known as 'googleMapDirectionServiceFunctions.route') into its own object (for neatness's sake, wasn't really required). Hopefully doesn't cause too much headache with the conflicts. If it wasn't changed between last time and now, maybe just replacing would be easier.
Anyway, hope this helps.
